core/mo-services-impl/nmf-platform-generic-impl/src/main/java/esa/mo/platform/impl/provider/gen/AutonomousADCSAdapterInterface.java

```diff
@@ -50,7 +50,7 @@
    * @throws UnsupportedOperationException if the implementation does not support the requested ADCS
    * mode
    */
-  void setDesiredAttitude(AttitudeMode attitude) throws IOException, UnsupportedOperationException;
+  void setDesiredAttitude(AttitudeMode attitude, org.ccsds.moims.mo.platform.autonomousadcs.structures.AttitudeDeterminationMode attitudeDeterminationMode) throws IOException, UnsupportedOperationException;
```

Comment:
No corresponding change on the Simulator adapter, which is in the same repo, but it might not be needed (see other comments).

Reply:
Can't you use "AttitudeDeterminationMode" directly and add the rest in the imports?

Reply:
I deleted this change since the atitudeDeterminationMode is now part of the AttitudeMode structure